beets/importer.py

```diff
@@ -1469,6 +1469,16 @@ def group(item):
 MULTIDISC_PAT_FMT = br'^(.*%s[\W_]*)\d'
 
 
+def is_subdir_of_any_in_list(path, dirs):
+    """Returns True if path os a subdirectory of any directory in dirs
+    (a list). In other case, returns False.
+    """
+    for d in dirs:
+        if path.startswith(d + b'/'):
+            return True
+    return False
```

Comment:
That would be the other way around,
```
any(d in ancestry(path) for d in dirs)
```
```path``` would be ```/.../album/CD1/covers``` and dirs would be the list of already included paths to collapse, so it would include ```/.../album/CD1```